### PR TITLE
Likes added

### PR DESCRIPTION
**Pull request by Malavikaexp**
Likes

### CODE REVIEW

File: src/BookYourShow.Api/Controllers/LikeController.cs [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BookYourShow.Controllers
+{
+    [Route("[controller]")]
```
Comment (basilin) [inline src/BookYourShow.Api/Controllers/LikeController.cs]: follow api standards mentioned in the doc

File: src/BookYourShow.Api/Controllers/LikeController.cs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    [ApiController]
+    public class LikeController : ControllerBase
+    {
+        //Constructor Dependency Injection for LikeRepo
```
Comment (basilin) [inline src/BookYourShow.Api/Controllers/LikeController.cs]: remove

File: src/BookYourShow.Api/Controllers/LikeController.cs [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        public async Task<IActionResult> GetLike()
+        {
+
+            var like = await likeRepository.GetLike();
+            if (like == null)
+            {
+                return NotFound();
+            }
+            return Ok(like);
+
```
Comment (basilin) [inline src/BookYourShow.Api/Controllers/LikeController.cs]: remove unwanted space

File: src/BookYourShow.Api/Controllers/LikeController.cs [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+        }
+
+        [HttpGet("{id}")]
+        [ProducesResponseType(typeof(Likes), 200)]
+        [ProducesResponseType(404)]
+        //[TypeFilter(typeof(CustomExceptionFilter))]
```
Comment (basilin) [inline src/BookYourShow.Api/Controllers/LikeController.cs]: remove

**Review by basilin — CHANGES_REQUESTED** [review]

File: src/BookYourShow.Api/Controllers/LikeController.cs (hunk 1 above)
Comment (Malavikaexp) [reply src/BookYourShow.Api/Controllers/LikeController.cs]: Comments resolved

File: src/BookYourShow.Api/Controllers/LikeController.cs (hunk 3 above)
Comment (Malavikaexp) [reply src/BookYourShow.Api/Controllers/LikeController.cs]: comment resolved

File: src/BookYourShow.Api/Controllers/LikeController.cs (hunk 4 above)
Comment (Malavikaexp) [reply src/BookYourShow.Api/Controllers/LikeController.cs]: comment resolved

**Pull request merged**